beep/structure.py

```diff
@@ -1183,18 +1198,27 @@ def determine_structuring_parameters(
                     hppc_rpt = ["reset", "hppc", "rpt_0.2C", "rpt_1C", "rpt_2C"]
                     hppc_rpt_len = 5
                     diagnostic_starts_at = [
-                        1, 1 + run_parameter["diagnostic_start_cycle"].iloc[0] + 1 * hppc_rpt_len,
+                        1,
+                        1
+                        + run_parameter["diagnostic_start_cycle"].iloc[0]
+                        + 1 * hppc_rpt_len,
                     ]
                     for i in range(1, 100):
                         diag_cycle_num = (
-                            i * (run_parameter["diagnostic_interval"].iloc[0] + hppc_rpt_len)
+                            i
+                            * (
+                                run_parameter["diagnostic_interval"].iloc[0]
+                                + hppc_rpt_len
+                            )
                             + 1
                             + run_parameter["diagnostic_start_cycle"].iloc[0]
                             + 1 * hppc_rpt_len
                         )
                         diagnostic_starts_at.append(diag_cycle_num)
                     diagnostic_available = {
-                        "parameter_set": run_parameter["diagnostic_parameter_set"].iloc[0],
+                        "parameter_set": run_parameter["diagnostic_parameter_set"].iloc[
```

Comment:
This is looks kind of strange... no problem with the code, but seems less readable

Reply:
I agree. As far as I can tell, it's an issue with how Black formats things.

I have not been impressed with how it splits long lines, and given that it's not configurable, I think we should consider using [yapf](https://github.com/google/yapf) instead.